example/glue/GlobalCollectorDelegate.hpp

```diff
@@ -136,7 +136,13 @@ class MM_GlobalCollectorDelegate
 	 * @param env environment for calling thread
 	 */
 	void prepareHeapForWalk(MM_EnvironmentBase *env) {}
-	
+
+	/* Read Barrier Verifier specific methods */
+#if defined(OMR_ENV_DATA64) && !defined(OMR_GC_COMPRESSED_POINTERS)
+	void poisonSlots(MM_EnvironmentBase *env) {}
```

Comment:
What are these APIs used for? I am surprised it has to be language specific and not a common OMR implementation.

Reply:
Beside dealing with (poisoning) heap slots, these methods can deal with roots slots, which are language dependent. In Open J9, it can specifically poison monitor table slots, class statics and JNI weak global, which are root candidates for Read Barrier (in fact, Monitor table slots are already relying on them).